src/transport/SecureSession.h

```diff
@@ -161,6 +161,10 @@ class SecureSession : public Session, public ReferenceCounted<SecureSession, Sec
             return GetMRPConfig().mIdleRetransTimeout * (CHIP_CONFIG_RMP_DEFAULT_MAX_RETRANS + 1);
         case Transport::Type::kTcp:
             return System::Clock::Seconds16(30);
+        case Transport::Type::kBle:
+            // TODO: Figure out what this should be, but zero is not the right
+            // answer.
+            return System::Clock::Seconds16(30);
```

Comment:
30 is too long. Please use 5 for now, and it can likely be trimmed down to a bit less even

Reply:
What is the sideffect of setting this too high? I imagine we time out slower, however for BLE this is commissioning only, so the difference would be "failed to commission" in 5 seconds vs 30. Would it be acceptable to just make it "very long" just in case?

Reply:
Alright.  Set to 5, and I'm doing some looking into what a not-just-guessed-at value would be.

Reply:
@andy31415 The side-effect of setting it too high is mostly that we end up taking longer to detect timeout when a device just doesn't respond to the "join network" command.  I think that's the main place where this value actually ends up mattering in practice.